crates/cxx-qt-lib/build.rs

```diff
@@ -154,6 +162,10 @@ fn main() {
     cxx_qt_lib_headers::write_headers(format!("{}/cxx-qt-lib", out_dir));
     builder.include(out_dir);
 
+    // Enable Qt Gui in C++ if the feature is enabled
+    #[cfg(feature = "qt_gui")]
```

Comment:
Do we need this in `cxx-qt-build` as that also has `cxx-qt-lib-headers`, but is that only for `common.h` etc ?

Maybe this is the point to split out `cxx-qt-gen` headers from `cxx-qt-lib` ones.

Reply:
Build uses whether the qt_module is enabled.